scripts/get-data.js

```diff
@@ -0,0 +1,63 @@
+const rimraf = require("rimraf");
```

Comment:
I'm not sure of the point of this script, given that we already have (bash) scripts to get data and narratives. If you need datasets which `npm run get-data` doesn't currently download then they can be added to [get-data.sh](https://github.com/nextstrain/auspice/blob/master/scripts/get-data.sh).

Note that #1050 may appear SARS-CoV-2 specific but it's actually a more general problem of being able to change datasets within a single narrative.